config/default/test/contact-summary/active_pregnancy.spec.js

```diff
@@ -0,0 +1,277 @@
+const { expect } = require('chai');
+const TestRunner = require('medic-conf-test-harness');
+const path = require('path');
+const moment = require('moment');
+const sinon = require('sinon');
+const { pregnancyRegistrationScenarios, pregnancyHomeVisitScenarios } = require('../form-inputs');
+const harness = new TestRunner({
+  xformFolderPath: path.join(__dirname, '../../forms/app'),
+});
+let clock = sinon.useFakeTimers(moment('2000-01-01').toDate());
```

Comment:
This assignment is unnecessary since it happens again in `beforeEach`.

Reply:
I tried removing the assignment but it caused a bunch of other tests (nools-extras.spec.js) to fail when run together. I think it has something to do with `clock.restore()` not doing its job in certain scenarios. I am investigating the problem and will update once solved.

Reply:
I guess this problem is because `restore()` does not work after calling `useFakerTimers()` twice in a row, as mentioned in this comment (also tested it): https://github.com/sinonjs/sinon/issues/550#issuecomment-339360964
Some effort will be required to ensure that the timers are restored every time after faking the timers before faking again.